website/docs/installation.md

```diff
@@ -2,26 +2,23 @@
 title: Installation
 ---
 
-The following instructions are for **Ubuntu 22.04** with the dependency versions supported as part of the [dev container](./contributing.md/#visual-studio-code-development-container) environment we use for builds and tests. For other distributions and OS, refer to the appropriate installation instructions for each of the components instead.
-
-1. Install [Go v1.19](https://go.dev/doc/install) or newer.
-
-2. Install the [make](https://www.gnu.org/software/make/) build tool.
-
-3. Install and start [Docker](https://www.docker.com/get-started/).
-
-4. Install [buildkit v0.10.5](https://github.com/moby/buildkit#quick-start) or newer. Alternatively, the `copa` CLI tool can also be used with the [buildkit container](https://hub.docker.com/r/moby/buildkit/tags/).
-
-5. Install [trivy v0.34.0](https://aquasecurity.github.io/trivy/latest/getting-started/installation/) or newer.
-
-6. Clone and make **copa**:
-
-   ```bash
-   sudo apt update
-   sudo apt install git
-   git clone https://github.com/project-copacetic/copacetic
-   cd copacetic
-   make
-   # OPTIONAL: install copa to a pathed folder
-   sudo mv dist/linux_amd64/release/copa /usr/local/bin/
-   ```
\ No newline at end of file
+## Homebrew
+On MacOS and Linux, `copa` can be installed via [Homebrew](https://brew.sh/):
+
+```bash
+brew install copa
+```
+
+## GitHub
+You can download the latest and previous versions of `copa` from the [GitHub releases page](https://github.com/project-copacetic/copacetic/releases).
+
+## Build from source
+`copa` can also be built directly from source:
```

Comment:
do we want to keep the requirements here?

Reply:
I think we just need to rationalize where this information shows up: My preference from other projects is that all build & test dependencies are specified in dev setup instructions (so docker and make should be mentioned here), as well as their minimum version requirements. Others like buildkit might be specified at point of use, such as in the [patching instructions](https://github.com/project-copacetic/copacetic/blob/main/docs/tutorials/patch.md).

Reply:
Maybe we just change that "Build from source" header to "Development Setup" and keep that section the same as it currently is in the docs?

Reply:
Yeah, I think that makes sense